include/substrait/common/NumberUtils.h

```diff
@@ -0,0 +1,20 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+#pragma once
+
+#include <iostream>
+
+namespace io::substrait::common {
+
+class NumberUtils {
+ public:
+  static bool is_number(const std::string& value) {
```

Comment:
It would be more flexible to accept `std::string_view`.

Reply:
Fixed